app/models/application_qualification.rb

```diff
@@ -65,6 +65,13 @@ def incomplete_degree_information?
     false
   end
 
+  def incomplete_gcse_information?
+    return true if award_year.nil?
+    return true if grade.nil? && structured_grades.nil?
```

Comment:
I think we can just change combine these two lines, it's pretty clear what's going on.

So `return true if award_year.nil? || ( grade.nil? && structured_grades.nil?)`

Reply:
I think we can also use the established pattern here of creating a constant 

EXPECTED_GCSE_DATA = %i[
    qualification_type
    subject
    award_year
  ].freeze

and then doing something like

`
  def incomplete_gcse_information?
     return true if grade.nil? && structured_grades.nil?
     return true if EXPECTED_GCSE_DATA.any? do |field_name|
      send(field_name).blank?
    end

    false
  end
`